src/test/java/org/terracotta/statistics/ratio/RatioTest.java

```diff
@@ -89,8 +89,8 @@ public void test_ratio_calculated() throws Exception {
       list = statisticsRegistry.findSampledStatistic("OnHeap:HitRatio").history(since);
       System.out.println(list.size() + " samples since " + since);
       since = System.currentTimeMillis();
-      for (Timestamped<? extends Number> timestamped : list) {
-        System.out.println(timestamped.getTimestamp() + " - " + timestamped.getSample());
+      for (Sample<?> sample : list) {
+        System.out.println(sample.getTimestamp() + " - " + sample.getSample());
       }
       try {
         Thread.sleep(1000);
```

Comment:
Can we get rid of the sleep?

Reply:
Not useful because this is linked to the way the old and deprecated `StatisticsRegistry` was working. This test will be removed if we can also remove the old and unused code.
The sleep time is linked to the sampling frequency.

Reply:
ok